test/java/brs/common/TestInfrastructure.java

```diff
@@ -2,7 +2,7 @@
 
 public class TestInfrastructure {
 
-  public static final String IN_MEMORY_DB_URL = "jdbc:h2:mem:test;";
+  public static final String IN_MEMORY_DB_URL = "jdbc:sqlite::memory:";
```

Comment:
good one... does it work? Did you run tests?

Reply:
I build releases with running tests - worked out